domain/model/state/modelstate.go

```diff
@@ -107,12 +108,15 @@ SELECT uuid,
 FROM model;
 `
 
-	var model coremodel.ReadOnlyModel
+	var (
+		controllerUUIDStr string
```

Comment:
This makes me shudder, the 90s called and wants its Hungarian notation back.

Reply:
Done & Done